app/views/info/_lead_metrics.html.erb

```diff
@@ -1,16 +1,33 @@
 <section id="lead-metrics">
   <% if lead_metrics %>
+    <%- if format == 'guide' %>
```

Comment:
I think the confusion that I went through could be avoided by having a boolean `is_multipart` flag instead of a `format` variable?